services/hanke-service/build.gradle.kts

```
@@ -156,6 +159,16 @@ tasks {
 			exceptionFormat = TestExceptionFormat.FULL
 		}
 	}
+
+	jacocoTestReport {
+		mustRunAfter("test", "integrationTest")
+		reports {
+			xml.required.set(true)
+		}
+		executionData.setFrom(
+            fileTree(buildDir).include("/jacoco/test.exec", "/jacoco/integrationTest.exec")
+        )
```

For some reason, this file uses tabs for indentation. These lines use spaces instead.
```suggestion
			fileTree(buildDir).include("/jacoco/test.exec", "/jacoco/integrationTest.exec")
		)
```

Or we could re-indent the whole file with spaces. Or we could make Spotless and ktfmt do the formatting for us by adding this under spotless.kotlin: `target("**/*.kt", "**/*.kts")`.

---

Added target("**/*.kt", "**/*.kts")

Pretty much the whole file is now changed, but I guess it is a good thing.

---

Yes, it changed the tabs to spaces, so it touched almost every line.